test/integration/host_js_test.rb

```diff
@@ -96,7 +96,7 @@ class HostJSTest < IntegrationTestWithJavascript
 
     test "edit page" do
       visit host_details_page_path(@host)
-      click_button 'Edit'
+      find('.host-details-header-section').find_button('Edit').click # the Edit button for the host details, not the host comment
```

Comment:
I think this is a regression introduced in 37093a54fd36ab247c809d86420ea341f4630ac7, but it's weird if we've been ignoring this test failure since May.

Reply:
something, something, doesn't always load quickly enough or whatever.

this test HAS been super flaky, which could be the regression here.

Reply:
https://github.com/theforeman/foreman/pull/10298

now to find out why the global params one is failing